src/main/java/com/skynest/clients/SkyNestBackendClient.java

```diff
@@ -2,16 +2,18 @@
 
 import com.skynest.constants.ApiConstants;
 import com.skynest.models.*;
+import io.restassured.http.ContentType;
 import io.restassured.response.Response;
 
+import java.io.File;
 import java.net.URISyntaxException;
 import java.util.UUID;
 
 import static org.apache.http.HttpHeaders.AUTHORIZATION;
 
 public class SkyNestBackendClient extends BaseClient {
-    private static String userId = "userId";
-    private static String bucketId = "bucketId";
+    private static String userIdKey = "userId";
```

Comment:
these can, and probably should be final as well given they're unlikely to change on the backend